diff --git a/keys b/keys
index 953d08b..31e1673 100644
--- a/keys
+++ b/keys
@@ -1,3 +1,4 @@
+# ~/.fluxbox/keys:
 # click on the desktop to get menus
 OnDesktop Mouse1 :HideMenus
 OnDesktop Mouse2 :WorkspaceMenu
@@ -35,8 +36,8 @@ OnTitlebar Move1  :StartMoving
 # middle click on the titlebar to lower
 OnTitlebar Mouse2 :Lower
 
-# right click on the titlebar for a menu of options
-OnTitlebar Mouse3 :WindowMenu
+# right click on the titlebar for resizing
+OnTitlebar Mouse3 :MacroCmd {StartResizing NearestCorner}
 
 # alt-tab
 Mod1 Tab :NextWindow {groups} (workspace=[current])



diff --git a/src/FbWinFrame.cc b/src/FbWinFrame.cc
index 937f87f..71508b6 100644
--- a/src/FbWinFrame.cc
+++ b/src/FbWinFrame.cc
@@ -945,7 +945,7 @@ void FbWinFrame::reconfigure() {
 
         // align handle and grips
         const int grip_height = m_handle.height();
-        const int grip_width = 20; //TODO
+        const int grip_width = m_window.width()/2; //TODO
         const int handle_bw = static_cast<signed>(m_handle.borderWidth());
 
         int ypos = m_window.height();

